addons/project/models/project.py

```diff
@@ -394,6 +394,11 @@ def _read_group_stage_ids(self, domain, read_group_order=None, access_rights_uid
     legend_done = fields.Char(related='stage_id.legend_done', string='Kanban Valid Explanation', readonly=True)
     legend_normal = fields.Char(related='stage_id.legend_normal', string='Kanban Ongoing Explanation', readonly=True)
 
+    # Next activity
+    next_activity_id = fields.Many2one(related="activity_log_ids.next_activity_id", string="Next Activity")
```

Comment:
@tde-banana-odoo Shouldn't we make something generic that comes with mail module instead of declaring these 3 fields in all modules ?